collector/sam/template.yaml

```diff
@@ -7,7 +7,7 @@ Resources:
     Properties:
       Code:
         S3Bucket: logzio-aws-integrations-us-east-1
-        S3Key: aws-security-hub-collector/0.0.1/function.zip
+        S3Key: aws-security-hub-collector/0.0.2/function.zip
```

Comment:
Shouldn't it hold the placeholders for the workflow to replace them when it triggeres?

Reply:
Applied.